### PR TITLE
Added remaining post related routes

### PR DESCRIPTION
**Pull request by AtharvaSune**
Added the remaining post related routes, functionality is still not complete, have to add middleware capabilities.

### CODE REVIEW

File: src/routes/posts.js [hunk 1]
```diff
@@ -80,10 +111,14 @@ router.get('/search', (req, res, next) => {
...
  * the comment to the Array
  */
-router.post('/:postid', (req, res, next) => {
-
-    Post.findById(req.params.postid)
-        .then(post => {
-            post.comments
-        })
+router.post('/:postid', Auth.isLoggedIn, (req, res, next) => {
+    let newComment = {
```
Comment (ameya0598) [inline src/routes/posts.js:115]: Can we add time and date to the comment as well?
What about the upvotes and downvotes?

**Review by ameya0598 — CHANGES_REQUESTED** [review]

File: src/routes/posts.js (hunk 1 above)
Comment (AtharvaSune) [reply src/routes/posts.js:115]: Date is set to Date.now() as default value, I will add upvotes, downvotes to the comment sub doc :+1:

**Comment by AtharvaSune** [comment]
Added upvotes and downvotes to post model, and set default value to 0

**Comment by ameya0598** [comment]
One more thing, we should also add a field which says *reacted?* to make sure that one person is allowed to upvote or downvote only once.

**Comment by AtharvaSune** [comment]
> One more thing, we should also add a field which says _reacted?_ to make sure that one person is allowed to upvote or downvote only once.

Right, so for that does this method seem like a good idea,  for posts we maintain a reacted array , Which consists list of people who have reacted to the post (list of `unique identifier`)

**Comment by ameya0598** [comment]
> > One more thing, we should also add a field which says _reacted?_ to make sure that one person is allowed to upvote or downvote only once.
> 
> Right, so for that does this method seem like a good idea, for posts we maintain a reacted array , Which consists list of people who have reacted to the post (list of `unique identifier`)

Yes

**Comment by AtharvaSune** [comment]
For now we'll just maintain who have reacted, once that functionality is tested, we can test more features like type od react and stuff like that
@ameya0598

**Review by ameya0598 — APPROVED** [review]

**Pull request merged**